tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -686,6 +687,16 @@ var _ = ginkgo.Describe("[tidb-operator] TiDBCluster", func() {
 		err = oa.WaitForTidbClusterReady(tc, 30*time.Minute, 15*time.Second)
 		framework.ExpectNoError(err, "Expected TiDB cluster scaled in and ready")
 	})
+
+	ginkgo.It("tkctl test cmds [use list] ", func() {
+		cluster := newTidbClusterConfig(e2econfig.TestConfig, ns, "restarter", "admin", "")
+		cluster.Resources["pd.replicas"] = "3"
+		cluster.Resources["tikv.replicas"] = "3"
+		cluster.Resources["tidb.replicas"] = "2"
+		oa.DeployTidbClusterOrDie(&cluster)
+		oa.CheckTidbClusterStatusOrDie(&cluster)
+		oa.CheckTkctlOrDie(&cluster)
```

Comment:
It's not necessary to add a method `CheckTkctlOrDie` on `operatorActions`. Can you move `CheckTkctlOrDie` implementation in this code block?